app/views/spina/registers/history.html.haml

```diff
@@ -19,18 +19,18 @@
         %span{itemprop: "name"} Register updates
 
 %main.container{role:'main'}
-  .grid-row
-    .column-two-thirds.js-hidden
-      %div{class: "js-filter-list", "aria-hidden" => "true"}
-        %form.js-filter-form
-          %fieldset
-            .form-group
-              %label{class: "form-label", for: "search"} Search for an update
-              %input{name: "search", id: "search", class: "form-control form-control-3-4"}
+  .search-records-wrapper#search_wrapper
+    .grid-row
+      = form_tag history_path(@register.slug), method: :get, id: 'search_updates' do
+        .column-one-third
+          .records-search
+            = label_tag 'Search updates', nil, class: 'form-label', for: 'q'
```

Comment:
Frontend is expecting param `q` but backend is expecting `search_input` so it does not work.

Reply:
The FE and BE were aligned, till someone change it. Now, everything is `q`, but still I think this name is meaningless.

Reply:
OK, but I would rather be consistent and wrong than inconsistent!

Reply:
+1 to what @gidsg said.